.github/workflows/Test.yml

```diff
@@ -16,671 +16,692 @@ jobs:
-      uses: actions/setup-python@v2
-      with:
-        python-version: 3.8
-        architecture: ${{ matrix.pyarch }}
-
-    - name: '🐍 Install Python dependencies'
-      run: |
-        pip3 install -r testsuite/requirements.txt
-
-    - name: '🚦 Test installation of pyGHDL through pip'
-      run: pip install ("git+https://github.com/ghdl/ghdl.git@" + $env:GHDL_HASH)
-
-    - name: '🚦 Test pyGHDL entrypoints'
-      run: |
-        ghdl-dom help
-        ghdl-ls --help
-
-    - name: '🚦 Test pyunit testsuite'
-      run: |
-        cd testsuite
-        python3 -m pytest -vsrA pyunit
+      - name: '🛠 Install package'
+        shell: msys2 {0}
+        run: |
+          pacman --noconfirm -U artifact/mingw-w64-${{ matrix.arch }}-ghdl-${{ matrix.pkg }}-*.zst
+
+      - name: '🛠 Set envvars'
+        run: |
+          $GHDL = (& msys2 -c 'cygpath -w /') + 'MINGW${{ matrix.bits }}\bin\ghdl.exe'
+          $GHDL_HASH = (& $GHDL version hash)
+          echo "GHDL_HASH=$GHDL_HASH" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
+          $GHDL_PREFIX = (& msys2 -c 'cygpath -w /') + 'MINGW${{ matrix.bits }}\lib\ghdl\'
+          echo "GHDL_PREFIX=$GHDL_PREFIX" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
+
+      - name: '🐍 Setup Python'
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+          architecture: ${{ matrix.pyarch }}
+
+      - name: '🐍 Install Python dependencies'
+        run: |
+          pip3 install -r testsuite/requirements.txt
+
+      - name: '🚦 Test installation of pyGHDL through pip'
+        run: pip install ("git+https://github.com/ghdl/ghdl.git@" + $env:GHDL_HASH)
+
+      - name: '🚦 Test pyGHDL entrypoints'
+        run: |
+          ghdl-dom help
+          ghdl-ls --help
+
+      - name: '🚦 Test pyunit testsuite'
+        run: |
+          cd testsuite
+          python3 -m pytest -vsrA pyunit
 
 #
 # Windows CPython pyGHDL Test with standalone zipfile and pyGHDL wheel
 #
 
   win-cpython-standalone:
+    name: '🚦🐍🥡 ${{ matrix.name }}'
+    runs-on: windows-latest
+
     needs:
       - win-generate-standalone-zip
       - pyGHDL
-    runs-on: windows-latest
+
     strategy:
       fail-fast: false
       matrix:
         include:
-        #- { pkg: '32-llvm', pyarch: x86 } ! not yet functional
-        #- { pkg: '64-llvm', pyarch: x64 }
-        #- { pkg: '32-mcode', pyarch: x86 } #! the tarball generation on MINGW32 needs to be fixed
-        - { pkg: '64-mcode', pyarch: x64 } #! simulation with mcode is not yet supported on win64
-    name: '🚦🐍🥡 ${{ matrix.pkg }}'
+#          - {name: 'llvm32', pkg: '32-llvm', pyarch: x86 } ! not yet functional
+#          - {name: 'llvm64', pkg: '64-llvm', pyarch: x64 }
+#          - {name: 'mcode32', pkg: '32-mcode', pyarch: x86 } #! the tarball generation on MINGW32 needs to be fixed
+          - {name: 'mcode64', pkg: '64-mcode', pyarch: x64 } #! simulation with mcode is not yet supported on win64
```

Comment:
If field `name` is added, better use `pkg` and `bits`, as in other jobs. So the "name" would be `${{ matrix.pkg }}${{ matrix.bits }}`.